### PR TITLE
Price documentation update

### PR DESCRIPTION
**Pull request by jcnelson**
Explicitly state that the price of a name/namespace is in satoshis or microstacks.

### CODE REVIEW

File: docs/api-specs.md [hunk 1]
```diff
@@ -765,6 +765,10 @@ This endpoint is used to get the price of a namespace, while explicitly
 indicating the cryptocurrency units.  This is because going forward, namespaces
 are not necessarily priced in Bitcoin.
 
+The `amount` given will be in the smallest possible units of the currency.  For
```
Comment (moxiegirl) [inline docs/api-specs.md]: > Specify the `amount` n the smallest possible units of the currency.  For
> `BTC`, the `amount` is in satoshis.  For `STACKS`, the `amount` is in
> micro-Stacks.

Technical documentation, unlike some other forms of writing, speaks to the reader in the present -- the user has a problem to solve and consults the documentation to find out what to do.

Comment (kantai) [reply docs/api-specs.md]: Today, new namespaces must be registered using the STACKS currency, so the text here for the price endpoints is correct, I think:

e.g., https://core.blockstack.org/v2/prices/namespaces/trytobuyme

returns:

```json
{
  "amount": "640000000", 
  "units": "STACKS"
}
```

File: docs/api-specs.md [hunk 2]
```diff
@@ -811,6 +815,10 @@ are not necessarily priced in Bitcoin.
 This endpoint is used to get the price of a name, denoted in a specific
 cryptocurrency (not necessarily Bitcoin).
 
+The `amount` given will be in the smallest possible units of the currency.  For
+`BTC`, the `amount` will be in satoshis.  For `STACKS`, the `amount` will be in
+micro-Stacks (1/10,000th of a Stack).
```
Comment (kantai) [inline docs/api-specs.md]: A microstack is `1e-6`th of a Stack, right ? That's (1/1,000,000th).

Comment (jcnelson) [reply docs/api-specs.md]: One hundred-thousandth.  We both done goofed.  1/100,000th

Comment (kantai) [reply docs/api-specs.md]: I always get confused by prefixes and have to check like 10 times.
1e-6 is one one-millionth (1 / 10^6) and matches the SI micro prefix (https://en.wikipedia.org/wiki/Micro-) and the blockstack-core constant (https://github.com/blockstack/blockstack-core/blob/master/blockstack/lib/config.py#L98).

Comment (jcnelson) [reply docs/api-specs.md]: yup, 1 millionth it is.

**Comment by kantai** [comment]
@jcnelson -- is this good to merge?

**Comment by jcnelson** [comment]
Yes -- will do so.

**Pull request merged**